src/univariate/continuous/generalizedpareto.jl

```diff
@@ -15,8 +15,9 @@ f(x; \\mu, \\sigma, \\xi) = \\begin{cases}
 ```
 
 ```julia
-GeneralizedPareto()             # Generalized Pareto distribution with unit shape and unit scale, i.e. GeneralizedPareto(0, 1, 1)
-GeneralizedPareto(μ, σ)         # Generalized Pareto distribution with shape ξ and scale σ, i.e. GeneralizedPareto(0, σ, ξ)
+GeneralizedPareto()             # Standard exponential distribution (GPD with shape 0, location 0, scale 1)
+GeneralizedPareto(ξ)            # Generalized Pareto distribution with shape ξ, location 0, and scale 1
```

Comment:
The choice for using `ξ` in the one-argument constructor seems a bit arbitrary, is there a specific reason for it? (Of course, the same could be said about the existing two-argument constructor).

Reply:
> The choice for using `ξ` in the one-argument constructor seems a bit arbitrary, is there a specific reason for it? (Of course, the same could be said about the existing two-argument constructor).

Parametrizing the Generalized Pareto in terms of scale and shape is pretty common. Parametrizing it only in terms of shape is less common but still done from time to time -- Wikipedia notes:
```
The generalized Pareto distribution (GPD) [...] is specified by three parameters: location {\displaystyle \mu }\mu , scale {\displaystyle \sigma }\sigma , and shape {\displaystyle \xi }\xi .[1][2] Sometimes it is specified by only scale and shape[3] and sometimes only by its shape parameter. 
```